tls/s2n_psk.h

```diff
@@ -56,6 +56,16 @@ struct s2n_psk {
     struct s2n_blob early_secret;
 };
 
+struct s2n_chosen_psk { 
+    struct s2n_external_psk external_psk;
+    uint16_t wire_index;
+};
+
+struct s2n_psk_identity {
+    uint8_t *data;
+    size_t length;
```

Comment:
Is length size_t, or is the length defined by the RFC as uint8_t, uint16_t, etc?